include/RAJA/operators.hxx

```diff
@@ -245,11 +245,11 @@ template <typename T>
 RAJA_HOST_DEVICE constexpr T min()
 {
   return 0;
-};
+}
 template <>
 RAJA_HOST_DEVICE constexpr int16_t min<int16_t>()
 {
-  return 0xFFFF;
+  return -32768;
```

Comment:
Later in the file, macros from float.h are used (like FLT_MAX). There are analogous macros for integral types defined in limits.h that would probably be more appropriate here.